data/repos.yml

```diff
@@ -270,6 +270,12 @@
   sentry_url: false
   dashboard_url: false
 
+- repo_name: docker-collectd-plugin
```

Comment:
Not sure this is part of DGU, I certainly haven't used it and it hasn't been touch in almost 2 years, maybe another repo to archive

Reply:
I'll take it out of this PR so it can be looked at separately.